### DISCUSSION TITLE
Cannot start weed server with options file

### DISCUSSION THREAD

**Discussion by franshst** [seaweedfs/seaweedfs#4364] [General]
Hi,
I am trying to get weed server starting with an options file. My command line is:
sudo weed server -options /data/seaweedfs/weed.toml
weed.toml contents
master.dir = "/data/seaweedfs"
dir = "/data/seaweedfs/v1"

weed server crashes with message:
F0401 23:16:28.209079 server.go:249 Check Meta Folder (-mdir="= "/data/seaweedfs"") Writable: stat = "/data/seaweedfs": no such file or directory

I am very sure that directory /data/seaweedfs exists.
It appears that the quotes are still around the string. How do I get rid of the quotes? They are needed in the toml file.

Thanks for any help
Frans

**Comment by chrislusf** [comment]
it's not supporting toml format.

**Comment by franshst** [comment]
Thanks for the hint. I was really confused because other configuration files are in toml format. I removed the quotes  and got it working now.
